### PR TITLE
510 measure playground

### PR DESCRIPTION
**Pull request by KaylaKremer**
#510 

Build: http://alpha.blueraster.io/gfw-mapbuilder/510-measure-playground/

### CODE REVIEW

File: src/js/components/Map.js [hunk 1]
```diff
@@ -153,21 +154,27 @@ export default class Map extends Component {
...
       } else {
-        if (map.infoWindow && map.infoWindow.getSelectedFeature) {
+        if (map.infoWindow && map.infoWindow.getSelectedFeature()) {
           const selectedFeature = map.infoWindow.getSelectedFeature();
+          console.log('selectedFeature', selectedFeature);
           if (selectedFeature && selectedFeature.geometry) {
             editToolbar.activate(Edit.EDIT_VERTICES, selectedFeature);
           }
         }
+        return;
```
Comment (SwampGuzzler) [inline src/js/components/Map.js]: @KaylaKremer Why the `return` statement?

Comment (KaylaKremer) [reply src/js/components/Map.js]: Accidentally left in. Will remove.

Comment (KaylaKremer) [reply src/js/components/Map.js]: Accidentally left in. Will remove.

Comment (SwampGuzzler) [reply src/js/components/Map.js]: @KaylaKremer Coolio - after that, let's resolve the **conflicts** and then merge this in

**Pull request merged**